packages/hardhat-core/scripts/issue-339.js

```diff
@@ -0,0 +1,41 @@
+const edr = require("@nomicfoundation/edr")
```

Comment:
@fvictorio should this file be kept? Feel free to modify this to be automatically testable